.travis.yml

```diff
@@ -19,6 +19,8 @@ os:
   - osx
 script:
   - npm test
-
+after_success:
+  - npm run test:coverage
+  - cat ./coverage/lcov.info | ./node_modules/codecov/bin/codecov
```

Comment:
Is this line necessary? This would limit us to only running on unix, right, since `cat` isn't available on Windows?

Is it possible to rewrite this to use `shx cat`? (should be `./lib/cli.js cat ./coverage/lcov.info` or something similar)